test/fixtures/experimental/import-meta/without-dynamic-import/actual.js

```diff
@@ -0,0 +1,4 @@
+const x = import.meta;
+const url = import.meta.url;
+import.meta;
+import.meta.url;
```

Comment:
Was there a reason this test didn't also have ` import.meta.couldBeMutable = true;`?